tests/integration/framework/process/exec/options.go

```diff
@@ -53,16 +55,17 @@ func WithExitCode(code int) Option {
 
 // WithEnvVars sets the environment variables for the command. Expects a list
 // of key value pairs.
-func WithEnvVars(envs ...string) Option {
+func WithEnvVars(keyValues ...string) Option {
 	return func(o *options) {
-		if len(envs)%2 != 0 {
-			panic("envs must be a list of key value pairs")
+		if len(keyValues)%2 != 0 {
+			panic("keyValues must be a list of key value pairs")
 		}
-		if o.envs == nil {
-			o.envs = make(map[string]string)
+		if o.envVars == nil {
+			o.envVars = make(map[string]string)
 		}
-		for i := 0; i < len(envs); i += 2 {
-			o.envs[envs[i]] = envs[i+1]
+		for i := 0; i < len(keyValues); i++ {
+			o.envVars[keyValues[i]] = keyValues[i+1]
+			i++
```

Comment:
```suggestion
```

Did you mean to put `i++` inside the for loop? If that was intentional, why not just use `i += 2`?